src/petals/server/throughput.py

```diff
@@ -18,6 +17,20 @@
 
 logger = get_logger(__file__)
 
+try:
+    import speedtest
+except ImportError:
+    logger.error("Please `pip install speedtest-cli==2.1.3`")
+    raise
```

Comment:
```suggestion
    raise ImportError("Please `pip install speedtest-cli==2.1.3`")
```

Reply:
Respectfully disagree: it is possible that speedtest-cli throws ImportError because some other module failed to import inside of it. In this case, it is best to let user know the specific error (via `raise`)

Reply:
nope, you were right
![image](https://user-images.githubusercontent.com/3491902/207843855-c5cdbd8a-b7b6-49c1-96d5-70516b37d2e0.png)